apps/e2e/integration/reconciliation_order.spec.ts

```diff
@@ -1,90 +1,243 @@
-import { test, expect } from "@playwright/test";
+import { expect } from "@playwright/test";
 
 import { baseURL } from "./constants";
-import { getDashboard } from "@/helpers";
+import { testOrders } from "@/helpers/fixtures";
+import { createSupplierOrder } from "@/helpers/cr-sqlite";
+import { getDbHandle } from "@/helpers";
 
-test.beforeEach(async ({ page }) => {
-	await page.goto(baseURL);
-	await getDashboard(page).waitFor();
-
-	page.getByLabel("Main navigation");
-	page.getByRole("listitem").last().click();
-	const nav = page.getByLabel("Main navigation");
-	await nav.waitFor();
-	await page.getByLabel("CreateReconciliationOrder").waitFor();
-	page.getByLabel("CreateReconciliationOrder").click();
-	await expect(page.getByLabel("CreateReconciliationOrder")).toBeDisabled();
-
-	await page.reload();
+// * Note: its helpful to make an assertion after each <enter> key
+// as it seems that Playwright may start running assertions before page data has fully caught up
+testOrders("should show correct initial state of reconciliation page", async ({ page, placedOrderLines }) => {
+	await page.goto(`${baseURL}orders/suppliers/orders/`);
 	await page.getByText("Ordered").nth(1).click();
-
 	await page.getByRole("checkbox").nth(1).click();
-
 	await page.getByText("Reconcile").first().click();
 
+	// Verify initial UI elements
 	await expect(page.getByText("Reconcile Deliveries")).toBeVisible();
+	await expect(page.getByPlaceholder("Enter ISBN of delivered books")).toBeVisible();
```

Comment:
Check:
- the empty state is there and not the table
- the Commit button is disabled
- the compare button at the bottom is not visible